components/webln.js

```diff
@@ -0,0 +1,106 @@
+import { createContext, createRef, useContext, useEffect, useImperativeHandle, useState } from 'react'
+
+const WebLNContext = createContext({})
+export const WebLNContextRef = createRef()
+
+const fetchWebLNProvider = async () => {
+  // sync provider from local storage
+  // TODO check if there is a better, more secure option
+  const configJson = window.localStorage.getItem('bc:config')
```

Comment:
If possible you should not directly access the storage like this, it's not part of the Bitcoin Connect public API

Reply:
I am only doing this because I couldn't get `onConnected` to fire on page load. I would love to not do this, haha

That's what I meant with "why do I have to do this myself?" in the TODO in `onConnected`:

> // TODO
//   why is this not fired on page load when there is a provider available in local storage?
//   why do i have to do this myself?

Reply:
Using `isConnected` now in e06e862